server/src/domain/auth/auth.service.ts

```diff
@@ -339,8 +339,12 @@ export class AuthService {
       }
 
       return new issuer.Client(metadata);
-    } catch (error: Error | any) {
-      this.logger.error(`Error in OAuth discovery: ${error}`, error?.stack);
+    } catch (error: any) {
+      if (error instanceof AggregateError) {
+        this.logger.error(`Error in OAuth discovery: ${error}`, error?.stack, error?.errors);
+      } else if (error instanceof Error) {
+        this.logger.error(`Error in OAuth discovery: ${error}`, error?.stack);
+      }
```

Comment:
I believe you can just change line 342 to be `error: any | AggregateError` and then use a single logger.error with `error?.errors`, instead of if/else with copy/pasted log lines.

Reply:
Original problem was that it wouldn't even find the AggregateError, but now it does, so fixed it to be like that instead.